project3/styles.css

```diff
@@ -0,0 +1,424 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+:root {
+    --color-blue: #163346;
+    --color-gray: #d1d5de;
+    --color-offwhite: #fff;
+    --color-lightgray: #cececd;
+    --color-pureblack: #000000;
+    --color-offblack: #1e2019;
+    --color-lime: #72f79f;
+
+    --font-teko: 'Teko', sans-serif;
+    --font-bench: 'BenchNine', sans-serif;
+
+}
+
+body {
+    background-color: var(--color-blue);
+    color: var(--color-offwhite);
+    font-family: var(--font-bench);
+    font-size: 22px;
+    margin: 0;
+}
```

Comment:
This stylesheet is starting off great; you have things in the proper order and have included a great and thorough `body` ruleset 👍